src/Illuminate/Console/Command.php

```diff
@@ -321,73 +333,111 @@ public function table(array $headers, $rows, $style = 'default')
     }
 
     /**
+     * Check if verbosity level is in bounds.
+     *
+     * @param  string|int  $level
+     * @return bool
+     */
+    public function inVerbosity($level)
+    {
+        if (isset($this->verbosity[$level])) {
+            $level = $this->verbosity[$level];
+        } elseif (! is_int($level)) {
+            $level = OutputInterface::VERBOSITY_NORMAL;
+        }
+
+        return $this->getOutput()->getVerbosity() >= $level;
+    }
+
+    /**
+     * Write a string as output.
+     *
+     * @param  string  $string
+     * @param  string|null  $style The output style of the string
+     * @param  int|string  $verbosity_level
+     * @return void
+     */
+    public function writeln($string, $style = null, $verbosity_level = OutputInterface::VERBOSITY_NORMAL)
+    {
+        if ($this->inVerbosity($verbosity_level)) {
```

Comment:
This was added in 2.8. 

https://github.com/laravel/framework/blob/5.1/composer.json#L32

https://github.com/symfony/symfony/blob/2.7/src/Symfony/Component/Console/Output/Output.php#L130-L132